native/src/main/java/io/ballerina/stdlib/serdes/Utils.java

```diff
@@ -21,29 +21,64 @@
 import io.ballerina.runtime.api.Environment;
 import io.ballerina.runtime.api.Module;
 import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.types.ArrayType;
 import io.ballerina.runtime.api.utils.StringUtils;
 import io.ballerina.runtime.api.values.BError;
 
+import java.util.Locale;
+
 /**
  * Utility functions of SerDes module.
  *
  * @since 0.1.0
  */
 public class Utils {
 
+    public static final String SERDES_ERROR = "Error";
     private static Module serdesModule = null;
 
-    public static void setModule(Environment env) {
-        serdesModule = env.getCurrentModule();
-    }
-
```

Comment:
Why is this method removed? If this is removed, the `getModule` method will not work.
The `setModule` method should be called from the package init method in the Ballerina code.

Reply:
Not removed function present in line 44